.env.example

```
@@ -26,19 +33,19 @@ REDIS_PORT=6379
 # Portal: https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps
 MSGRAPH_CLIENT_ID=your-azure-client-id-here
 MSGRAPH_TENANT_ID=your-azure-tenant-id-here
-MSGRAPH_CLIENT_SECRET=your-azure-client-secret-here
+MSGRAPH_CLIENT_SECRET=your_msgraph_client_secret_here  # Run: node scripts/generate-secure-secrets.js msgraph
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MSGRAPH_CLIENT_SECRET key should go before the MSGRAPH_TENANT_ID key

(UnorderedKey)